curriculum/challenges/english/25-front-end-development/quiz-css-typography/66ed9010f45ce3ece4053eb8.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --answer--
 
-Placeholder answer
+It allows you to use custom fonts by importing them
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which service offers a large selection of free fonts for use on websites?
 
 #### --distractors--
 
-Placeholder distractor 1
+Typekit
 
 ---
 
-Placeholder distractor 2
+Adobe Fonts
 
 ---
 
-Placeholder distractor 3
+Font Squirrel
 
 #### --answer--
 
-Placeholder answer
+Google Fonts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is it important to include fallback fonts in your CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+It reduces page load times
 
 ---
 
-Placeholder distractor 2
+It prevents browser compatibility issues
 
 ---
 
-Placeholder distractor 3
+It gives the page more visual variety
 
 #### --answer--
 
-Placeholder answer
+It ensures a consistent experience in case the custom font doesn’t load
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How many values does the `text-shadow` property usually have?
 
 #### --distractors--
 
-Placeholder distractor 1
+1
 
 ---
 
-Placeholder distractor 2
+2
 
 ---
 
-Placeholder distractor 3
+5
 
 #### --answer--
 
-Placeholder answer
+3 or 4
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What CSS property is used to set the font family for an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+`font-size`
 
 ---
 
-Placeholder distractor 2
+`font-weight`
 
 ---
 
-Placeholder distractor 3
+`text-align`
 
 #### --answer--
 
-Placeholder answer
+`font-family`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is typographic hierarchy important in design?
 
 #### --distractors--
 
-Placeholder distractor 1
+It improves the readability of large blocks of text
 
 ---
 
-Placeholder distractor 2
+It ensures consistency across different devices
 
 ---
 
-Placeholder distractor 3
+It allows decorative fonts to be used effectively
 
 #### --answer--
 
-Placeholder answer
+It helps organize content and guide users through the information
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is **not** a common font property?
 
 #### --distractors--
 
-Placeholder distractor 1
+`font-size`
 
 ---
 
-Placeholder distractor 2
+`font-weight`
 
 ---
 
-Placeholder distractor 3
+`font-style`
 
 #### --answer--
 
-Placeholder answer
+`font-padding`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a correct example of the `text-shadow` property in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`text-shadow: solid 5px black;`
 
 ---
 
-Placeholder distractor 2
+`text-shadow: 5px blur black;`
 
 ---
 
-Placeholder distractor 3
+`text-shadow: inset 2px 2px white;`
 
 #### --answer--
 
-Placeholder answer
+`text-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the first value in the `text-shadow` property control?
```

Comment:
Actually the first value, while optional, controls the color of the shadow. 

Source: https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow